python/graphstorm/model/utils.py

```diff
@@ -393,6 +393,23 @@ def _exchange_node_id_mapping(rank, world_size, device,
     # move mapping into CPU
     return gather_list[0].to(th.device("cpu"))
 
+def load_dist_nid_map(node_id_mapping_file, ntypes):
```

Comment:
Can we just rename the old function instead of adding new one that is just a wrapper?

Reply:
Same as here, let me make the function public.